components/SiteNavbar.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <nav class="flex flex-wrap flex-row-reverse justify-between border-b border-gray-300 px-5 py-3 lg:flex-row">
+    <div class="flex lg:hidden" @click="mobileView = !mobileView">
+      <button class="flex items-center border border-gray-300 rounded px-4 py-1 text-gray-600 hover:text-black hover:border-black hover:bg-gray-100">
+        <svg class="fill-current h-3 w-3" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg">
+          <title>Menu</title>
+          <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"/>
+        </svg>
+      </button>
+    </div>    
+    <div class="flex flex-grow">
+      <h1 class="w-full py-2 text-lg text-center font-semibold leading-6 tracking-tight lg:text-left">speedrun.website</h1>
+    </div>
+    <div class="flex">
+      <svg xmlns="http://www.w3.org/2000/svg" class="h-10 w-10 mx-1 lg:hidden" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+        <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M12 8v4l3 3m6-3a9 9 0 11-18 0 9 9 0 0118 0z" />
+      </svg>
+    </div>
+    <div class="flex flex-col w-full my-2 text-center font-semibold lg:flex-row lg:w-auto lg:my-0" v-if="mobileView || lg">
+      <a class="rounded px-3 py-2 m-1 text-gray-600 lg:my-0 hover:bg-gray-100 hover:text-gray-800" href="#">Home</a>
```

Comment:
Not blocking and not necessary for now, but we probably should consider making those links basic components

Reply:
Agreed as well, although I think these nav items may change fairly quickly in the near future, and this initial PR is just to get in some structure that aligns closely with the wireframes. This is something we may want to revisit after a few more layout revisions (or alternatively, confirmation of a standard).